hail/python/hailtop/batch/backend.py

```diff
@@ -763,15 +763,16 @@ async def compile_job(job):
 
         jobs_to_command = {j.id: cmd for j, cmd in jobs_to_command.items()}
 
+        deploy_config = get_deploy_config()
+        url = deploy_config.external_url('batch', f'/batches/{batch_handle.id}')
+        print(f'Submitted batch {batch_handle.id}, see {url}')
```

Comment:
Let's hide this behind an `if not wait` because the progress bar will include the URL when we're waiting